tests/unit/Serializers/ItemSerializerTest.php

```diff
@@ -25,7 +25,7 @@ protected function buildSerializer() {
 		$claimsSerializerMock->expects( $this->any() )
 			->method( 'serialize' )
 			->will( $this->returnCallback( function( Claims $claims ) {
-				if ( $claims->isEmpty() ) {
+				if ( count( $claims ) === 0 ) {
```

Comment:
I would prefer `empty()`, but since "this is only a test"<sup>TM</sup> it doesn't matter much.